packages/language-server/server/completion/completion-item-kind.js

```diff
@@ -1,74 +1,59 @@
+const { ObjectType } = require("@hegel/core/type-graph/types/object-type");
+const { FunctionType } = require("@hegel/core/type-graph/types/function-type");
 const { CompletionItemKind } = require("vscode-languageserver");
-
-const FUNCTION_MATCHER = "=>";
-const CLASS_MATCHER = "class";
-const CONSTRUCTOR_MATCHER = "Constructor";
-
-const kinds = [
-  {
-    itemKind: CompletionItemKind.Function,
-    matcher: FUNCTION_MATCHER,
-  },
-  {
-    itemKind(type) {
-      return type.isConstant
-        ? CompletionItemKind.Constant
-        : CompletionItemKind.Variable;
-    },
-    matcher(type) {
-      return (
-        !String(type.name).includes(CLASS_MATCHER) &&
-        !String(type.name).includes(FUNCTION_MATCHER) &&
-        !String(type.name).includes(CONSTRUCTOR_MATCHER) &&
-        type.properties === undefined &&
-        !type.isVariableTypeAlias
-      );
-    },
-  },
-  {
-    itemKind: CompletionItemKind.Class,
-    matcher(type) {
-      return (
-        (String(type.name).includes(CLASS_MATCHER) ||
-          type.properties !== undefined) &&
-        !String(type.name).includes(CONSTRUCTOR_MATCHER)
-      );
-    },
-  },
-  {
-    itemKind: CompletionItemKind.Constructor,
-    matcher: CONSTRUCTOR_MATCHER,
-  },
-  {
-    itemKind: CompletionItemKind.Interface,
-    matcher(variableInfo) {
-      return variableInfo.isVariableTypeAlias;
-    }
-  }
-];
+const {
+  CONSTRUCTABLE,
```

Comment:
Could you sort imports identifiers by a name length, please?

Reply:
Oh, sorry, I forgot to sort(